### ISSUE TITLE
Need a way to collapse left panel for mobile

### ISSUE THREAD

**Issue by maddadder** [qTox/qTox#6275]
I am using this software on mobian. My screen resolution is 720x1440. I cannot read my chat messages without rotating the screen. I need a way to collapse left panel for mobile view.

**Comment by Chiitoo** [comment]
Can you not drag it to the left from around the border?

**Comment by maddadder** [comment]
![signal-2020-12-14-003851](https://user-images.githubusercontent.com/441667/102029113-0ce27d00-3da5-11eb-9ef6-6b0fd432aa8f.jpeg)
> Can you not drag it to the left from around the border?

I can, but it has a min width setting that prevents me from collapsing it to the point that it becomes a button (to allow re-expansion upon click). I want to point out that the device I am using is this: https://www.pine64.org/pinephone/. Not android or ios. See attached.

**Comment by Chiitoo** [comment]
Oh right, sorry, I'm perhaps a little too used to the 'Multiple windows mode' where it's possible to completely hide that part in the chat windows.

I wonder if that mode would be usable there?  I guess it might get a bit tight, too.

**Comment by sudden6** [comment]
Definitely an interesting use case you have here. The only way I see to get this to work is using multiple windows mode as Chiitoo suggested, but I'm not sure if your window manager handles this gracefully.

**Comment by maddadder** [comment]
using multiple windows mode:

![signal-2020-12-14-192034](https://user-images.githubusercontent.com/441667/102125570-cc850c80-3e41-11eb-92fb-cd28feac0e7a.jpeg)

**Comment by maddadder** [comment]
I'd be willing to donate a https://www.pine64.org/pinephone/ to whomever wants to work on this.

**Comment by Chiitoo** [comment]
Hmmm, you should be able to drag that line to the left to make that section with the username completely hidden, though I'd not be surprised if you were unable to grab it properly seeing what is going on there (and if that is the case, I wonder if flipping the thing sideways (if it adjusts the elements like mobile thingies usually do) would help with that).

**Comment by maddadder** [comment]
Yes I have tried changing the orientation. I think a code change is required.

**Comment by Chiitoo** [comment]
Just to make sure, are you able to drag it any direction at all?

It's normal that when being dragged to the left, it stops until the thing that is dragging it reaches around half point (in my set-up at least), and going beyond that the area will plop into hiding.

That would of course require you to be able to grab the thing in the first place...

**Comment by anthonybilinski** [comment]
I see the same behaviour of @Chiitoo. If I drag up to half way to the left, it does nothing, but past there it snap to totally hidden. Does this work for you, @charlierlee?

I think that bar shouldn't be there are all in the multiple window mode.. but that's for another time.

**Comment by maddadder** [comment]
@anthonybilinski  Can you send a screen capture of that? I don't have that on the phone, and I don't have that on my desktop either. On desktop I am using: qTox version v1.16.3-2build1-debian. I can't tell you what version I have on mobian because the UI has set a fixed min width, and I can't get to the about page.

**Comment by sudden6** [comment]
@charlierlee can you start qTox from a terminal on the phone? it should print the version at startup or I think with the `-v` parameter

1.16.3 is quite old, maybe that's why it's different.

**Comment by maddadder** [comment]
@sudden6 I loaded the nightly build and still have the same problem

**Comment by anthonybilinski** [comment]
@charlierlee example video here:

https://user-images.githubusercontent.com/10469203/106948448-92691600-66e0-11eb-9ab6-613c9f25e69a.mp4

(file is actually a webm not mp4, but renamed to get around github file type restrictions).

**Comment by Chiitoo** [comment]
Nice cursor pointer thing.  :]

All this also reminds me how I want to also work on a tabbed interface still (tabs as in a web-browser, not as they currently are in qTox).
